### PR TITLE
Setup Codeowners File

### PR DESCRIPTION
**Pull request by thebeanogamer**
Set up a CODEOWNERS file to automatically require a review from a member of the server staff on pull requests.

This means that PRs will effectively receive two rubber stamps, one from server staff, and one from Community Developers who work on the bot.

As I am not sure how this responds to teams being used, I have set it to use myself, although I am going to play around with teams if the PR will allow me to do so, so apologies for the spammy emails.

### CODE REVIEW

**Comment by jos-b** [comment]
You can use teams. Just make a team and call it with @cyberdiscovery/team_name.

**Comment by jos-b** [comment]
neverMIND You ALREADY changED IT

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# To require the approval of the server staff as well as that of the technical community developer team
+*       @CyberDiscovery\server-admin-team
```
Comment (jos-b) [inline CODEOWNERS]: This should be `/`
```suggestion
*       @CyberDiscovery/server-admin-team
```

**Comment by thebeanogamer** [comment]
Yeah, the only thing I don't know is if it requires the entire team or just one person from it?

**Comment by thebeanogamer** [comment]
I was going to email support, but if you know, that'd be easier

**Comment by jos-b** [comment]
I believe it only requires one person from the team.

**Comment by thebeanogamer** [comment]
_I see you ghosting @AnonGuy_

**Comment by thebeanogamer** [comment]
I would override the merge restrictions to get this in and test it, but that does feel like it misses the point.

**Comment by jos-b** [comment]
Okay, looks like this does actually request approval from the whole team, but I assume that just means that it awaits one approval from the team?

**Comment by jos-b** [comment]
Only one way to find out ¯\_(ツ)_/¯

**Comment by thebeanogamer** [comment]
What's the worst that could happen other than all the terrible things that could happen?

**Comment by jos-b** [comment]
Also, if you accept my suggestion I can approve this

**Comment by thebeanogamer** [comment]
I feel judged for my use of the clearly inferior OS

**Comment by thebeanogamer** [comment]
And whilst you do have the technical knowledge for approving review ~which I seem to lack~, your contributions aren't quite sufficient yet.
image

**Comment by jos-b** [comment]
that logging handler was truly spectacular though 😃

**Comment by thebeanogamer** [comment]
@jos-b It was, and once #145 has been merged, I will finish #143 to add Sentry.io to the logging

**Comment by jos-b** [comment]
sentry.io let's go :muscle:

**Comment by thebeanogamer** [comment]
@joker314 I've made the change in repo settings. Adding the Community Developer team to the file would be counterproductive. The repo already requires 2 reviews from people with merge permissions, and all adding that would do is increase the amount of emails the bot team get.

**Pull request merged**